src/Riok.Mapperly/Descriptors/MappingBodyBuilders/BuilderContext/MembersMappingBuilderContext.cs

```diff
@@ -140,6 +140,11 @@ private void AddUnmatchedIgnoredTargetMembersDiagnostics()
     {
         foreach (var notFoundIgnoredMember in _ignoredUnmatchedTargetMemberNames)
         {
+            if (notFoundIgnoredMember.Contains("."))
```

Comment:
Use the `StringMemberPath.PropertyAccessSeparator` constant (same for the source members).